app/views/course/assessment/submission/submissions/_submissions.html.slim

```diff
@@ -0,0 +1,12 @@
+table.table.submissions
+  thead
+    tr
+      th = t('.student_name')
+      th = t('.status')
+      th = t('.grade')
+      th = t('.experience_points')
+      th
+  tbody
+    - course_users.each do |course_user|
```

Comment:
@weiqingtoh How about not load the `@course_users`, but directly access the course_user through: `submission.course_user`, I think that's the initial way you proposed ?  You can still order the submission by course_user names